shells/android/java/arcs/storage/service/IStorageService.aidl

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.storage.service;
+
+import arcs.storage.parcelables.ParcelableModelUpdate;
+import arcs.storage.parcelables.ParcelableOperations;
+import arcs.storage.parcelables.ParcelableSyncRequest;
+import arcs.storage.service.IResultCallback;
+import arcs.storage.service.IStorageServiceCallback;
+
+/**
+ * Exposed API for the StorageService.
+ *
+ * TODO: subject to change.
+ */
+interface IStorageService {
+    /**
+     * Registers an {@link IStorageServiceCallback} with the StorageService and returns its callback
+     * token.
+     */
+    int registerCallback(IStorageServiceCallback callback);
+
+    /** Unregisters the callback associated with the given {@param token}. */
+    void unregisterCallback(int token);
+
+    /** Sends the StorageService a SyncRequest ParcelableProxyMessage. */
```

Comment:
I don't know what the right convention is for .aidl files. I'd guess you should use a code font, like Javadoc. So ideally `{@link ParcelableProxyMessage}`, or otherwise `{@code ParcelableProxyMessage}`.

Throughout this file, and the other .aidl files too.

Also I'd probably just point to `ParcelableSyncRequest` here.

Reply:
Will-do when I set to working on these things for real